.github/workflows/release.yml

```diff
@@ -143,6 +143,8 @@ jobs:
           github_cache_dir="/__w"
           sudo mkdir -p "$github_cache_dir"
           sudo chmod -R 777 "$github_cache_dir"
+      - name: Install python3-config
+        run: sudo apk add python3-dev
```

Comment:
`python3-config` is called during the build, by `semgrep-core/src/cli-bridge/dune`.  Within this job, the `scripts/install-alpine-semgrep-core` script runs the build.  I'd be happy to put this info into a comment, but I refrained from doing so here (other than what the name says) since there aren't explanations of other steps.